src/Vendr.uSync/Handlers/VendrSyncHandlerBase.cs

```diff
@@ -75,6 +77,25 @@ public virtual IEnumerable<uSyncAction> ProcessPostImport(string folder, IEnumer
             return postActions;
         }
 
+        /// <summary>
+        ///  if there is a 'OneWay' setting in the config, then we will only import something
+        ///  if it doesn't already exist. 
+        /// </summary>
+        /// <remarks>
+        ///  On the vendr base class means, it can be applied to any of the handler configs.
+        /// </remarks>
+        protected override bool ShouldImport(XElement node, HandlerSettings config)
+        {
+            if (config.Settings.ContainsKey("OneWay") && config.Settings["OneWay"].InvariantEquals("true"))
+            {
+                // only import if it doesn't already exist.
+                var item = GetFromService(node.GetKey());
+                return item == null;
+            }
+
+            return false;
```

Comment:
Should the default be `true`?

ie, if `OneWay == false` or isn't present, it should just allow the import

Reply:
yes it should! I need to do some checks tommorow.

Reply:
yep, changed it to return base.ShouldImport - which is better because if we add base config options then this will still work with them too.